mmv1/products/compute/Interconnect.yaml

```diff
@@ -89,7 +89,6 @@ properties:
       URL of the InterconnectLocation object that represents where this connection is to be provisioned.
     resource: 'InterconnectLocations'
     imports: 'selfLink'
-    required: true
```

Comment:
Is making `location` optional in this resource intended? Sorry, I'm not fully understand what remote_location field is. Is location field not needed for resource creation in some cases?

Reply:
Yes, Incase of cross_cloud interconnect we don't need to specify location field as location field implies for google owned location, Incase of cross_cloud interconnect we have to replace it with remote_location field.

Reply:
Got it. Would you mind also clarifying it in the description of the field? Thanks!

Reply:
Done